browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2410,8 +2410,13 @@ L.Control.JSDialogBuilder = L.Control.extend({
 				button.setAttribute('aria-haspopup', true);
 
 			var buttonImage = L.DomUtil.create('img', '', button);
-			var icon = data.icon ? data.icon : builder._createIconURL(data.command);
-			L.LOUtil.setImage(buttonImage, icon.split('/').pop(), builder.map);
+			if (data.image) {
```

Comment:
I think we need also .icon   for  integrator's custome icons in notebookbar shortcuts bar:
https://github.com/CollaboraOnline/online/blob/f203a4d68f017a4c578deca317f559549a1b06c3/browser/src/control/Control.Notebookbar.js#L326

You can check that in the postMessage tester I think